src/prefect/client/schemas/objects.py

```diff
@@ -110,6 +110,7 @@ class StateDetails(PrefectBaseModel):
     pause_key: str = None
     run_input_keyset: Optional[Dict[str, str]] = None
     refresh_cache: bool = None
+    retriable: bool = None
```

Comment:
This is a change from the original PR which was setting `retriable` to `True` by default.